package.json

```diff
@@ -35,8 +35,8 @@
     "css-lint-docs": "stylelint --syntax scss \"site/docs/**/assets/scss/*.scss\" && stylelint \"site/docs/**/*.css\"",
     "css-lint-vars": "fusv scss/ site/docs/",
     "css-minify": "npm-run-all --parallel css-minify-*",
-    "css-minify-main": "cleancss --level 1 --format breaksWith=lf --source-map --source-map-inline-sources --output dist/css/bootstrap.min.css dist/css/bootstrap.css && cleancss --level 1 --format breaksWith=lf --source-map --source-map-inline-sources --output dist/css/bootstrap-grid.min.css dist/css/bootstrap-grid.css && cleancss --level 1 --format breaksWith=lf --source-map --source-map-inline-sources --output dist/css/bootstrap-reboot.min.css dist/css/bootstrap-reboot.css",
-    "css-minify-docs": "cross-env-shell cleancss --level 1 --format breaksWith=lf --source-map --source-map-inline-sources --output site/docs/$npm_package_version_short/assets/css/docs.min.css site/docs/$npm_package_version_short/assets/css/docs.min.css",
+    "css-minify-main": "csso --input dist/css/bootstrap.css --no-restructure --source-map file --input-source-map auto --output dist/css/bootstrap.min.css && csso --input dist/css/bootstrap-grid.css --no-restructure --source-map file --input-source-map auto --output dist/css/bootstrap-grid.min.css && csso --input dist/css/bootstrap-reboot.css --no-restructure --source-map file --input-source-map auto --output dist/css/bootstrap-reboot.min.css",
+    "css-minify-docs": "cross-env-shell csso --input site/docs/$npm_package_version_short/assets/css/docs.min.css --no-restructure --source-map file --input-source-map auto --output site/docs/$npm_package_version_short/assets/css/docs.min.css",
```

Comment:
`--input-source-map auto` is by default, no need to specify it

Reply:
Yeah, I know, but I was trying to understand why we have that difference in the source maps, without success so far.